dbfit-java/oracle/src/main/java/dbfit/environment/OracleStoredProcedureCall.java

```diff
@@ -15,13 +15,13 @@
 
 import dbfit.util.Direction;
 
-public class OracleStoredProcedureCall extends DbStoredProcedureCall {
+public class OracleStoredProcedureCall extends DefaultDbStoredProcedureCall {
```

Comment:
We're _extending_ the **default** in order to get a **non-default** implementation. It used to be that way before, just now the naming of the base class made it more obvious.
Perhaps that shouldn't bother us much right now - unless it helps to resolve some of the other questions.